src/partials/order.html

```diff
@@ -1,3 +1,42 @@
-<section class="order">
-
-</section>
\ No newline at end of file
+<section class="order-section">
+  <div class="order-form container">
```

Comment:
After tag `section ` should be `<div class="order-container container">`. In container should be added all your code.

Reply:
fixed